packages/service-provider-sfa/src/SfaServiceProvider.ts

```diff
@@ -90,6 +100,29 @@ class SfaServiceProvider extends ServiceProviderBase {
     return this.postboxKey;
   }
 
+  async getHostURL() {
+    return this.torusNodeEndpointOrigin;
+  }
+
+  async _delete1of1Key(endpoint?: string, enableLogging?: boolean) {
```

Comment:
delete1of1 in sfa no need accept endpoint.
it should only use the sfa's private endpoint